providers/common.rb

```diff
@@ -0,0 +1,71 @@
+include_recipe 'git'
```

Comment:
Does this actually work now in Chef 12? Cool.

You should probably use [this](http://www.fewbytes.com/using-include_recipe-in-chef-lwrp/) method for including recipes for Chef 11 compatibility though.